### PR TITLE
@craigspaeth: Money fields on SaleArtwork

### PR DESCRIPTION
**Pull request by dzucconi**
Included accounting, which should make it easy to throw in currency conversions.

I think this includes some breaking changes so I'll get that sorted in Force tomorrow and rebased with master here. Let me know if there's something I need to do to coordinate with Prediction, however.

### CODE REVIEW

File: npm-shrinkwrap.json [hunk 1]
```diff
@@ -2703,6 +2708,11 @@
       "from": "number-is-nan@>=1.0.0 <2.0.0",
       "resolved": "https://registry.npmjs.org/number-is-nan/-/number-is-nan-1.0.0.tgz"
     },
+    "numeral": {
+      "version": "1.5.3",
+      "from": "numeral@latest",
+      "resolved": "https://registry.npmjs.org/numeral/-/numeral-1.5.3.tgz"
+    },
```
Comment (dzucconi) [inline npm-shrinkwrap.json]: I'm going to use this on the follow and other counts, though that's not included in this PR

File: schema/types/formatted_number.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    if (ast.kind !== Kind.STRING || ast.kind !== Kind.INT) {
+      const error = `Query error: Can only parse strings and ints, got a: ${ast.kind}`;
+      throw new GraphQLError(error, [ast]);
+    }
+
+    return ast.value;
+  },
+});
+
+export default FormattedNumber;
```
Comment (dzucconi) [inline schema/types/formatted_number.js]: This is for numerals so we can return an Int (`1000`) or a formatted String (`"1,000"`)

File: test/schema/sale_artwork.js [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+              amount: '€2,000',
+            },
+            highest_bid: {
+              cents: 325000,
+              amount: '€3,250',
+            },
+            current_bid: {
+              cents: 325000,
+              amount: '€3,250',
+            },
```
Comment (dzucconi) [inline test/schema/sale_artwork.js]: Passes through the symbols that are returned in the response, if there's no symbol present then it just defaults to $

File: schema/sale_artwork.js [hunk 4]
```diff
@@ -32,46 +35,78 @@ const SaleArtworkType = new GraphQLObjectType({
...
               type: GraphQLString,
             },
             created_at: date,
-            amount_cents: {
-              type: GraphQLString,
+            is_cancelled: {
+              type: GraphQLBoolean,
+              resolve: ({ cancelled }) => cancelled,
+            },
+            amount: amount(({ amount_cents }) => amount_cents),
```
Comment (craigspaeth) [inline schema/sale_artwork.js]: Nice I like the consistency this is bringing to the sale artwork schema. I'm happy to make the Prediction changes—they're really minimal right now anyways b/c we just started integrated Metaphysics.

**Comment by craigspaeth** [comment]
:clap: Very cool! This should be super helpful for client apps and I like the consistency of the `is_` fields and `amount` args! Looks like you have a conflict, otherwise feel free to merge when ready. Great work as always!

**Comment by dzucconi** [comment]
Alright—the existing active bids module still functions normally with some deprecated fields; making this safe to merge. (I'll update that as well)

**Pull request merged**